packages/rest/src/rest-component.ts

```diff
@@ -40,16 +40,15 @@ export class RestComponent implements Component {
   };
   servers: {
     [name: string]: Constructor<Server>;
+  } = {
+    RestServer: RestServer,
```

Comment:
It can be further simplified as {RestServer}. See https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Object_initializer.